config/initializers/octokit.rb

```diff
@@ -0,0 +1,7 @@
+stack = Faraday::RackBuilder.new do |builder|
+  builder.response :logger
```

Comment:
I'm going to look at the server logs and see if this is something I want turned on.